services/acm/src/main/java/software/amazon/awssdk/services/acm/AcmAsyncClient.java

```diff
@@ -0,0 +1,1805 @@
+import software.amazon.awssdk.services.acm.model.DeleteCertificateRequest;
+import software.amazon.awssdk.services.acm.model.DeleteCertificateResponse;
+import software.amazon.awssdk.services.acm.model.DescribeCertificateRequest;
+import software.amazon.awssdk.services.acm.model.DescribeCertificateResponse;
+import software.amazon.awssdk.services.acm.model.ExportCertificateRequest;
+import software.amazon.awssdk.services.acm.model.ExportCertificateResponse;
+import software.amazon.awssdk.services.acm.model.GetAccountConfigurationRequest;
+import software.amazon.awssdk.services.acm.model.GetAccountConfigurationResponse;
+import software.amazon.awssdk.services.acm.model.GetCertificateRequest;
+import software.amazon.awssdk.services.acm.model.GetCertificateResponse;
+import software.amazon.awssdk.services.acm.model.ImportCertificateRequest;
+import software.amazon.awssdk.services.acm.model.ImportCertificateResponse;
+import software.amazon.awssdk.services.acm.model.ListCertificatesRequest;
+import software.amazon.awssdk.services.acm.model.ListCertificatesResponse;
+import software.amazon.awssdk.services.acm.model.ListTagsForCertificateRequest;
+import software.amazon.awssdk.services.acm.model.ListTagsForCertificateResponse;
+import software.amazon.awssdk.services.acm.model.PutAccountConfigurationRequest;
+import software.amazon.awssdk.services.acm.model.PutAccountConfigurationResponse;
+import software.amazon.awssdk.services.acm.model.RemoveTagsFromCertificateRequest;
+import software.amazon.awssdk.services.acm.model.RemoveTagsFromCertificateResponse;
+import software.amazon.awssdk.services.acm.model.RenewCertificateRequest;
+import software.amazon.awssdk.services.acm.model.RenewCertificateResponse;
+import software.amazon.awssdk.services.acm.model.RequestCertificateRequest;
+import software.amazon.awssdk.services.acm.model.RequestCertificateResponse;
+import software.amazon.awssdk.services.acm.model.ResendValidationEmailRequest;
+import software.amazon.awssdk.services.acm.model.ResendValidationEmailResponse;
+import software.amazon.awssdk.services.acm.model.UpdateCertificateOptionsRequest;
+import software.amazon.awssdk.services.acm.model.UpdateCertificateOptionsResponse;
+import software.amazon.awssdk.services.acm.paginators.ListCertificatesPublisher;
+import software.amazon.awssdk.services.acm.waiters.AcmAsyncWaiter;
+
+/**
+ * Service client for accessing ACM asynchronously. This can be created using the static {@link #builder()} method.
+ *
+ * <fullname>Certificate Manager</fullname>
+ * <p>
+ * You can use Certificate Manager (ACM) to manage SSL/TLS certificates for your Amazon Web Services-based websites and
+ * applications. For more information about using ACM, see the <a
+ * href="https://docs.aws.amazon.com/acm/latest/userguide/">Certificate Manager User Guide</a>.
+ * </p>
+ */
+@Generated("software.amazon.awssdk:codegen")
+@SdkPublicApi
+@ThreadSafe
+public interface AcmAsyncClient extends AwsClient {
```

Comment:
Will this cause duplicate class errors in IDEs? Should we disable codegen for this module?

Reply:
Maybe there way a better way, but I disabled codegen by deleting the `src/main/resources/codegen-resources` from these modules.

Reply:
Oh, okay. It could be done in the pom.xml, but just deleting it works too since this is temporary.